constants/sprite_constants.asm

```diff
@@ -28,7 +28,7 @@
 	const SPRITE_ROCKET                  ; $18
 	const SPRITE_CHANNELER               ; $19
 	const SPRITE_WAITER                  ; $1a
-	const SPRITE_ERIKA                   ; $1b
```

Comment:
Please add `SPRITE_ERIKA` to the legacy macros, like Gen 2 `SPRITE_BUENA`.

Reply:
Will do! Almost forgot sorry!

Reply:
Never mind, Gen 1 doesn't have a scripting language so there aren't legacy commands that need supporting.